src/help-widget-renderer.ts

```diff
@@ -1,23 +1,17 @@
-require('./styles/widget-styles.scss');
-require('./styles/omnibar-style-adjustments.scss');
+import './styles/omnibar-style-adjustments.scss';
+import './styles/widget-styles.scss';
 
 const IFRAME_ID: string = 'bb-help-iframe';
 const IFRAME_TITLE: string = 'BB Help';
 const IFRAME_SRC: string = 'https://host.nxt.blackbaud.com/bb-help/';
-const BB_HEADER_COLOR: string = '#71bf43';
-const BB_HEADER_TEXT_COLOR: string = '#fff';
+const BB_HEADER_COLOR: string = '#71bf43'; // 'rgb(113, 191, 67)';
+const BB_HEADER_TEXT_COLOR: string = '#fff'; // 'rgb(255, 255, 255)';
```

Comment:
Do we need the rgb color values for anything?

Reply:
Yes unfortunately. Some browsers translate the javascript.style into RGB values.  This is for a reference to those values.